tests/test_python_pr_unit.py

```
@@ -0,0 +1,433 @@
+
+    def test_python_version_enum_does_not_contain_none(self):
+        enum = load_schema()["properties"]["python_version"]["enum"]
+        assert None not in enum
+
+    def test_python_version_type_is_string(self):
+        field = load_schema()["properties"]["python_version"]
+        assert field["type"] == "string"
+
+
+# ---------------------------------------------------------------------------
+# python-imagestream.yaml – new 3.14 image stream entries
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)